src/main/java/org/cryptomator/cryptofs/health/dirid/LooseDirFile.java

```diff
@@ -0,0 +1,42 @@
+package org.cryptomator.cryptofs.health.dirid;
+
+import org.cryptomator.cryptofs.VaultConfig;
+import org.cryptomator.cryptofs.health.api.DiagnosticResult;
+import org.cryptomator.cryptolib.api.Cryptor;
+import org.cryptomator.cryptolib.api.Masterkey;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static org.cryptomator.cryptofs.health.api.CommonDetailKeys.DIR_ID_FILE;
+
+public class LooseDirFile implements DiagnosticResult {
```

Comment:
Please keep the convention of calling this a `DirIdFile`, since it always has been this way and causes confusion to rename it.

Reply:
Changed in [82775e7](https://github.com/cryptomator/cryptofs/pull/138/commits/82775e7e4fce91350a0b461bf4512126a2b88b42)